docs/servo_power_test/index.html

```diff
@@ -17,6 +17,7 @@
     <button id='request-serial-button'>Start</button>
     <button id='halt-button'>Cancel</button>
     <button id='download-button'>Download</button>
+    <span>number of iterations: <input id="iteration-input" type="number" placeholder="1" style="width: 32px;"></span>
```

Comment:
I think all usage of `style` attribute should be in .css files to separate the concerns, as said in the style guide:
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns